test/perf/compiled_extensions.py

```diff
@@ -996,6 +996,82 @@ def getattr(self):
         self.row_long.x
         self.row_long.y
 
+    @test_case
+    def getattr_repeats(self):
+        row = self.row
+        row.a
```

Comment:
I think a plain for here would accomplish the same since Python has no optimization on a for